File: API/Helpers/UserParams.cs
﻿namespace API.Helpers
{
    public class UserParams : PaginationParams
    {
        public string? Gender { get; set; }
        public string? CurrentUsername { get; set; }
        public int MinAge { get; set; } = 18;
        public int MaxAge { get; set; } = 80;
        public string? Country { get; set; }
        public string? City { get; set; }
        public string OrderBy { get; set; } = "lastActive";

    }
}

File: client/src/app/shared/models/user-params.ts
export class UserParams {
  gender: string = 'all';
  minAge: number = 18;
  maxAge: number = 79;
  pageNumber: number = 1;
  pageSize: number = 5;
  country: string = '';
  city: string = '';
  orderBy: string = 'lastActive';
}

export class PaginationParams {
  pageNumber: number = 1;
  pageSize: number = 5;
}
